nautobot/extras/tests/test_filters.py

```diff
@@ -365,15 +364,15 @@ class CustomFieldChoiceFilterSetTestCase(FilterTestCases.FilterTestCase):
 
     generic_filter_tests = (
         ["value"],
-        ["custom_field", "custom_field__name"],
+        ["custom_field", "custom_field__label"],
```

Comment:
Hm, might want to discuss this one. Should it be based around `slug` rather than `label`?